packages/constants/package.json

```diff
@@ -26,6 +26,7 @@
   },
   "dependencies": {
     "@latitude-data/compiler": "workspace:^",
+    "@opentelemetry/semantic-conventions": "^1.22.0",
```

Comment:
How big is this thing?

constants should be as clean and lean as possible. Don't put here big dependencies 🙏